src/app/routes/api/v3/admin/forms/admin-forms.form.routes.ts

```diff
@@ -229,3 +230,34 @@ AdminFormsFormRouter.get(
   '/:formId([a-fA-F0-9]{24})/verified-sms/count/free',
   AdminFormController.handleGetFreeSmsCountForFormAdmin,
 )
+
+AdminFormsFormRouter.route('/feedback')
+  /**
+   * Submit an admin form creating feedback
+   * and returns the feedback document on success
+   * @precondition user should be logged in
+   * @precondition Joi validation should enforce shape of req.body before this handler is invoked.
+   * @security session
+   *
+   * @returns 200 if feedback was successfully saved
+   * @returns 400 when Joi validation fails
+   * @returns 401 when user does not exist in session
+   * @returns 422 when user of given id cannnot be found in the database
+   * @returns 500 if database error occurs
+   */
+  .post(AdminFeedbackController.handleSubmitAdminFeedback)
+
+AdminFormsFormRouter.route('/feedback/:feedbackId([a-fA-F0-9]{24})')
+  /**
+   * Update an existing admin feedback
+   * @precondition Joi validation should enforce shape of req.body before this handler is invoked.
+   * @security session
+   *
+   * @returns 200 if feedback was successfully updated
+   * @returns 400 when Joi validation fails
+   * @returns 401 when user does not exist in session
+   * @returns 404 when admin feedback cannnot be found in the database
+   * @returns 422 when user of given id cannot be found in the database
```

Comment:
is this still valid?

Reply:
omg yes, its removed, thanks for pointing it out. [8405543](https://github.com/opengovsg/FormSG/pull/6454/commits/84055434476aa416b0af672dc402825f3c6ae14d) we no longer throw 422